hugo/content/1-basics/installation.md

```diff
@@ -0,0 +1,43 @@
++++
+next = "/1-basics/quick-tour"
+prev = "/1-basics/what-is-mangrove"
+title = "Installation"
+toc = true
+weight = 2
+
++++
+
+To get Mangrove up and running in your project, simply follow these instructions to install Mangrove on your system.
+
+## Prerequisites
+
+1. OS X or any other standard 'nix platform.
```

Comment:
I guess the only requirements are that you have one of the compilers I listed, and that your platform is little endian. Should I just make this "Any little endian platform"? I didn't include windows because we're not providing MSVC support yet.